test/step-definitions/responses.ts

```diff
@@ -65,8 +66,30 @@ export class responses {
   @then(/the response should contain:/)
   public dataResponseTable(table: { rawTable: [] }) {
     const data = this.context.tableToObject(table);
-    Object.keys(data).forEach((key) =>
-      expect(JSON.parse(this.context.response.text)[key]).to.eql(data[key]),
-    );
+    Object.keys(data).forEach((key) => {
+      if (data[key] === 'TYPE:ID') {
+        this.context.documentId = JSON.parse(this.context.response.text)[key];
```

Comment:
Is the `documentId` returned from the post request? I think that `documentId` should be used to check against  the response